lib/computer.rb

```diff
@@ -6,41 +6,55 @@ class Computer
   def initialize(name, size = 10)
     @board = Board.new(name, size)
     @ships = []
+    # TO DO: ^ why needed?
```

Comment:
Again, this relates to scope. Any time we declare a class variable with @ we need to initialize it in, well, the initializer. That's what that function is for.